### ISSUE TITLE
Buttons in top-right covered by login notification

### ISSUE THREAD

**Issue by Huh** [Huh/PopR_MT#20]
Move the buttons or see if you can hide the signed in notification.

**Comment by Huh** [comment]
Login notification removed using: 

```css
/* Hide shiny disconnect div  */
.shiny-server-account {
  display: none;
}
```

A new issue was made on the discussion issue tracker to allow this to show in a new div.

**Comment by ThatMikeBoy** [comment]
Closer to 9:30

- Bob Inman

On Apr 4, 2019 5:57 PM, Josh Nowak <notifications@github.com> wrote:

Closed #20<https://gcc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FHuh%2FPopR_MT%2Fissues%2F20&data=02%7C01%7Cbobinman%40mt.gov%7C6590157fd9704e25330508d6b9595607%7C07a94c98f30f4abbbd7ed63f8720dc02%7C0%7C0%7C636900190676079981&sdata=UDKTHLaOBz1%2B7pRGnJXRqewIqipaNyokRLbiSiMRdHc%3D&reserved=0>.

**Comment by Huh** [comment]
Sounds good
